components/automate-gateway/handler/authz.go

```diff
@@ -345,17 +345,23 @@ func (a *AuthzServer) getAllowedProjects(
 	// Fetches the id of the current user PLUS the team ids for that user
```

Comment:
🤔 Shouldn't the entire IAM v2[.1] business live in [automate-gateway/handler/iam/v2beta](https://github.com/chef/automate/tree/master/components/automate-gateway/handler/iam/v2beta)? What do you think?

Reply:
I agree; I may save that for a separate PR, though.

Reply:
😄 Would fit quite nicely under this PR's description though 😉 But yeah, it would involve changing the UI as well...